semantic/test/Semantic/Stat/Spec.hs

```diff
@@ -92,7 +93,7 @@ case_render_tags = do
 
 case_sendstat_delivers_datagram :: Assertion
 case_sendstat_delivers_datagram = do
-  client@StatsClient{..} <- defaultStatsClient
```

Comment:
We weren’t using any of these fields, I take it? That’s a funny wart that can creep in.